NodeJS/src/RestClient.ts

```diff
@@ -400,4 +425,48 @@ export class RestClient {
         return -error.response.status
       })
   }
+
+  private buildQueryParams(request: AddressBookSearchRequest) {
```

Comment:
Should unit test this method.

Reply:
Also this method or function could be defined in AddressBookSearchRequest interface.

Reply:
ok, will do

Reply:
done